pkg/yurthub/kubernetes/serializer/serializer.go

```diff
@@ -35,30 +41,173 @@ import (
 // YurtHubSerializer is a global serializer manager for yurthub
 var YurtHubSerializer = NewSerializerManager()
 
+// UnsafeDefaultRESTMapper is used to check whether the GVK is in the scheme according to the GVR information
+var UnsafeDefaultRESTMapper = NewDefaultRESTMapperFromScheme()
+
+func NewDefaultRESTMapperFromScheme() *meta.DefaultRESTMapper {
+	scheme := scheme.Scheme
+	defaultGroupVersions := scheme.PrioritizedVersionsAllGroups()
+	mapper := meta.NewDefaultRESTMapper(defaultGroupVersions)
+	// enumerate all supported versions, get the kinds, and register with the mapper how to address
+	// our resources.
+	for _, gv := range defaultGroupVersions {
+		for kind := range scheme.KnownTypes(gv) {
+			scope := meta.RESTScopeNamespace
```

Comment:
I think that scopes for all gvk are `meta.RESTScopeNamespace` is not make sense.

Reply:
Code is already updated, thanks for review